### DISCUSSION TITLE
Should I just give up? It's been more than a month since I raised a reinstatement ticket and I haven't got a single response from support

### DISCUSSION THREAD

**Discussion by Kareem-dev** [community/community#131264] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

As the title states, I raised a reinstatement ticket more than a month ago (ticket ID: 2827874) regarding the sudden and random suspension of my account (@Kareem01001011) for no apparent reason. I raised this reinstatement ticket as soon as I was suspended but I haven't gotten a single response ever since I raised the ticket. I made a discussion post 7 days after the suspension and I was basically told that I'll have to wait until they respond, but this is getting a little out of hand now. I really need to at least get back my username and starred repositories, but it seems like they're gone forever.

Should I just give up on my account and my entire work? Is ignoring my request and my need for the account back normal? I've done everything I can and nothing has been done yet. This has taken too long and I don't think GitHub will do anything about it and this is just too infuriating.

**Comment by gaomingzhao666** [comment]
Hey bro.

I clicked the link to your original account, and the page is displayed as 404.

I think you can still work on the current account and report that to the GitHub team, today is Sunday so maybe next week the GitHub team will respond to you.

Good luck bro, if the comment helped you or can solve your question please mark this comment as an answer. 😊

**Reply by Kareem-dev** [reply]
The page is displayed as 404 because the account is suspended. The least I want is to get my username back 🥲.

I'll try waiting a bit more to see if the GitHub team would give me any updates at least but honestly, I'm losing hope.

Thanks for your comment!

**Reply by gaomingzhao666** [reply]
Don't be sad bro, send more ticket reports to `The GitHub team` may work.

Anyway good luck bro😌

**Comment by Yewzir** [comment]
Hi Kareem

My word of advice is to not give up just yet. Recently, I found myself in an issue of my own for which I also raised a ticket with GitHub Support a few days back. Eventually, an employee had committed to my case and provided me with additional measures and guiding information while not looking down on you and staying respectful for the duration of the case. At first, I figured it'd be a lost cause because who knows what the current count of pending cases is or how they might be determining priority levels. Nevertheless, I was pleasantly surprised by their sign of life and I can honestly say that I will not be hesistating to contact them in the future, not by a long shot.

Spreaded around the place are mentions from GitHub staff in which they explicitly inform the community about exceptional delays in particular cases. For example, I encountered the following [statement](https://github.com/github/docs/issues/33722) in the list of issues for GitHub Docs.

> We may be slow to review contributions from June 28th through July 8th, as many people on the docs team will be taking time off.

Bear in mind that GitHub has a substantial amount of employees from all over the world and considering we find ourselves at the beginning of summer holidays, I'm positive that a really high number of those employees are travelling with their families as we speak. Unfortunately, while your co-workers may be perfectly capable of filling in for a decent number of your daily routines, there will always be that time where edge cases are pushed aside and waiting for the return of certain employees with the "master case knowledge", so to say.

Take care, pal.
Hicham

**Comment by Mee-meet** [comment]
Subject: Request for Unsuspension of GitHub Account

Dear GitHub Support Team,

I hope this message finds you well. My name is Meet Mehta, and I am writing to request the unsuspension of my GitHub account. The details of my account are as follows:

Username: MeetInCode
Email: meets.studentlife@gmail.com
I recently discovered that my account has been suspended, and I am unsure of the reason for this action. I rely heavily on GitHub for my studies and projects, and losing access to my repositories has caused significant disruption.

If the suspension was due to a misunderstanding or an error on my part, I sincerely apologize and am willing to take any necessary corrective actions to ensure compliance with GitHub's terms of service.

I would greatly appreciate your assistance in reviewing my case and providing guidance on how to resolve this issue. Thank you for your understanding and support.

Looking forward to your prompt response.

Best regards,
Meet Mehta
